packages/locale/src/translations/en.json

```diff
@@ -438,7 +438,9 @@
                 "birth_date": "Date of birth",
                 "errors": {
                     "isRequired": "{fieldName} is required",
-                    "isInvalid": "{fieldName} is invalid"
+                    "isInvalid": "{fieldName} is invalid",
+                    "PAYMENT_SOURCE_INFO_CANNOT_BE_VERIFIED": "The combination of your name and address could not be validated. Please correct your data and try again. You can find further information in the <a href='https://www.ratepay.com/en/ratepay-data-privacy-statement/' target='blank'>Ratepay Data Privacy Statement</a> or you can contact Ratepay using this <a href='https://www.ratepay.com/en/contact/' target='blank'>contact form.</a>",
```

Comment:
It's better use camelcase for the error key in all language files

Reply:
Also we have other languages in checkout js project:

<img width="242" alt="Screenshot 2023-10-16 at 13 50 47" src="https://github.com/bigcommerce/checkout-js/assets/25133454/7c5c2aa1-98cb-4f30-8e2b-47ad015e49e3">

Reply:
Ratepay used only in English and Germany languages